index.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title></title>
+	<title>lass</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<header class="container">
 		<div id="logo">My Logo</div>
```

Comment:
- [ ] Kindly use `<a>` tag instead of `div` tag, it's more semantic so when the user clicks your logo switch him to your home page.